sections/featured-product.liquid

```diff
@@ -215,11 +209,11 @@
                             name="options[{{ option.name | escape }}]"
                             form="{{ product_form_id }}"
                           >
-                            {%- for value in option.values -%}
-                              <option value="{{ value | escape }}" {% if option.selected_value == value %}selected="selected"{% endif %}>
-                                {{ value }}
-                              </option>
-                            {%- endfor -%}
+                            {% render 'product-variant-options',
+                              product: product,
+                              option: option,
+                              block: block
+                            %}
                           </select>
                           {% render 'icon-caret' %}
                         </div>
```

Comment:
We're missing the setting here, if we are to keep it, similar than what we have on `main-product.liquid` to choose `none` or `disabled` for `select_soldout_type`. 

@edmund-teh do you think we should offer that setting to merchant ? Basically it's a setting that only applies when the variant type is `dropdown` and here are the two visual output: 
- when set to none: [screenshot](https://screenshot.click/23-28-mq406-w621s.png)
- when set to disabled: [screenshot](https://screenshot.click/23-23-61xxa-10i5h.png)

Maybe it's good to keep both options for folks using apps to manage pre orders or out of stocks orders. 🤔